src/components/headerArea.scss

```diff
@@ -0,0 +1,160 @@
+.header-area {
+  padding: var(--contentPaddingPrimary);
+}
+
+.top {
+  display: flex;
+  border-top: 1px solid #000;
+  border-bottom: 1px solid #000;
+  padding: 24px 0 96px 0;
```

Comment:
I'm noticing `24px` and multiples of this are starting to show up a lot. I think these might be good candidates to think about adding as spacer variables.